src/AppBundle/Controller/DefaultController.php

```diff
@@ -5,17 +5,75 @@
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use \Pusher;
+use Symfony\Component\HttpFoundation\Session\Session;
+
+
 
 class DefaultController extends Controller
 {
     /**
-     * @Route("/", name="homepage")
+     * @Route("/start-game", name="start-game")
+     */
+    public function startGame(Request $request)
```

Comment:
For now it's okay.

Later, will have to put all code at proper place. Eg. DefaultController not a place for this.